web/build.gradle

```diff
@@ -192,7 +192,6 @@ dependencies {
 
     compile "se.inera.intyg.schemas:schemas-contract:${schemasContractVersion}"
     compile "se.inera.intyg.clinicalprocess.healthcond.rehabilitation:intyg-clinicalprocess-healthcond-rehabilitation-schemas:${schemasRehabilitationVersion}"
-    compile "se.inera.intyg.clinicalprocess.healthcond.srs:intyg-clinicalprocess-healthcond-srs-schemas:${srsSchemasVersion}"
```

Comment:
Varför ta bort beroendet, wc har väl fortfarande dependency till kontraktstyperna, som i SrsApiController ?

Reply:
Jag har flyttat EgenBedomningRiskType till rätt xsd och då behövdes inte längre beroendet till det schemat. 
Jag la till det i förra vändan för att få med den typen, men nu behövdes det inte längre så vitt jag kan se så jag tog bort det igen för att inte ha onödiga beroenden.
Oklart varför den behövdes tidigare, men förmodligen för att typen då låg i fel xsd. Nu bör beroendet komma in via infra som i sin tur har beroende till srs-schema.

Reply:
Kolla gärna på den här också, då den måste göras först för att inte bygget skall fallera https://github.com/sklintyg/schemas/pull/28